scripts/benchmark/duckduckgo/add-sentry-to-duckduckgo.patch

```diff
@@ -0,0 +1,66 @@
+diff --git a/build.gradle b/build.gradle
+index 3dcf39810f..be03ac46f0 100644
+--- a/build.gradle
++++ b/build.gradle
+@@ -14,10 +14,10 @@ buildscript {
+         mavenCentral()
+     }
+     dependencies {
+-        classpath 'com.android.tools.build:gradle:4.1.3'
++        classpath 'com.android.tools.build:gradle:7.0.2'
+         classpath "com.diffplug.spotless:spotless-plugin-gradle:$spotless"
+         classpath "com.squareup.anvil:gradle-plugin:$anvil_version"
+-
++        classpath "io.sentry:sentry-android-gradle-plugin:3.0.0-beta.3"
+         // NOTE: Do not place your application dependencies here; they belong
+         // in the individual module build.gradle files
+     }
+@@ -28,6 +28,25 @@ allprojects {
+         google()
+         mavenCentral()
+     }
++
++}
++
++subprojects {
++     apply plugin: 'io.sentry.android.gradle'
```

Comment:
Does it make sense to have the script bump the version? Or to use a locally installed package so we can run this in CI against a freshly built version? I guess the question should be: do you plan to run it in CI?

Reply:
Solved it using includedBuild for the plugin (uses local version) and dynamic version for the main SDK.

As for running it on CI I will give it a shot, but don't wanna spend much time now